src/st/dai/background_block_init.c

```diff
@@ -11,5 +11,5 @@ ObjInit2 OVL_EXPORT(BackgroundBlockInit)[] = {{
     .palette = PAL_NONE,
     .drawFlags = DRAW_DEFAULT,
     .drawMode = DRAW_TPAGE,
-    .animFrames = anim_frames,
+    .animFrames = (u8*)anim_frames,
```

Comment:
The animFrames element in BackgroundBlockInit requires a cast, but the cast is not needed when sending data to AnimateEntity.